src/js/claims-status/utils/appeals-v2-helpers.jsx

```diff
@@ -544,92 +544,92 @@ export function getEventContent(event) {
   switch (event.type) {
     case EVENT_TYPES.claimDecision:
       return {
-        title: 'VBA sent the original claim decision to you',
+        title: 'VA sent you a claim decision',
         description: '',
       };
     case EVENT_TYPES.nod:
       return {
-        title: 'VBA received your Notice of Disagreement',
+        title: 'VA received your Notice of Disagreement',
         description: '',
       };
     case EVENT_TYPES.droHearing:
-      return {
+      return { // TODO: does this need to be updated?
```

Comment:
@gnakm should this also be updated?

Reply:
This can be removed.

Reply:
Thanks @gnakm, this event (droHearing) has been removed.